tests/unit/core/utils/test_index.js

```diff
@@ -792,4 +797,81 @@ describe(__filename, () => {
       });
     });
   });
+
+  describe('getPreviewImage', () => {
+    it('returns the first full image from the previews array', () => {
+      const fullImage = 'https://addons.cdn.mozilla.net/full/12345.png';
```

Comment:
Can we use `${config.get('amoCDN')}` instead of `https://addons.cdn.mozilla.net` everywhere?